www/less/widenImages.less

```diff
@@ -4,31 +4,9 @@
     display: block;
     margin-left: auto;
     margin-right: auto;
-}
-
-@media (orientation: portrait) {
-    .wideImageOverride {
-        width:100% !important;
-        max-width:100% !important;
-        height:auto !important;
-    }
-
-    /* Tablet override */
-    @media (min-device-width: 768px) {
-        .wideImageOverride {
-            width:60% !important;
```

Comment:
Perhaps you'll get the same effect if you just change 60% to 100% here... or just remove lines 17 to 23 leaving the rest...